packages/aws-cdk/bin/cdk.ts

```diff
@@ -230,6 +230,8 @@ async function initCommandLine() {
         } else {
           return await cliInit(args.TEMPLATE, language);
         }
+      case 'version':
+        return print(version.DISPLAY_VERSION);
```

Comment:
`print` emits to STDERR while `--version` emits to STDOUT. You should use `data` instead.